glean.vcxproj

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need this file? Tell me more. I though it was generated by the compilation scripts.